### PR TITLE
add link to storage info

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9674

### CODE REVIEW

File: Teams/manage-whiteboard.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Users can share a whiteboard to make it available to all participants in a Teams
 
 To turn the Whiteboard app on or off, see [Enable Microsoft Whiteboard for your organization](https://support.office.com/article/1caaa2e2-5c18-4bdf-b878-2d98f1da4b24). Keep in mind that this setting will enable or disable Whiteboard for your entire organization, and not just for Teams.
 
+Whiteboards will be created in the OneDrive for Business folder of the person who starts the whiteboard. For more information, see [Manage data for Microsoft Whiteboard](/microsoft-365/whiteboard/manage-data-organizations?view=o365-worldwide#onedrive-for-business-storage-overview).
```
Comment (JohanFreelancer9) [inline Teams/manage-whiteboard.md:33]:
```suggestion
Whiteboards will be created in the OneDrive for Business folder of the person who starts the whiteboard. For more information, see [Manage data for Microsoft Whiteboard](/microsoft-365/whiteboard/manage-data-organizations).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@wlibebe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**